aspnetapp/Dockerfile.nano

```diff
@@ -0,0 +1,5 @@
+FROM microsoft/dotnet:nanoserver
```

Comment:
I think this should be a versioned image and it shouldn't rely on an sdk image.  I think 1.1-runtime-nanoserver would be more appropriate.